typescript/cli/src/deploy/core.ts

```diff
@@ -284,11 +279,6 @@ async function executeDeploy({
   const ismContracts: ChainMap<{ multisigIsm: DeployedIsm }> = {};
   const defaultIsms: ChainMap<Address> = {};
   for (const ismOrigin of chains) {
-    if (artifacts[ismOrigin].multisigIsm) {
```

Comment:
you can pass in IsmConfig and it should be able to recover the multisigIsm based on the validators you give

checkout https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2928

Reply:
actually, not necessary right now. Approved!